### PR TITLE
Feat/aut 2651/furigana ruby plugin

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/AUT-2651

**For test use https://github.com/oat-sa/tao-core/pull/3644** 

How to test:
- go to item or passage Authoring
- put interaction or text block
- put text
- select text
- use furigana plugin
- save item

image

### CODE REVIEW

**Review by marpesia — APPROVED** [review]
LGTM

**Review by Silvia-loza — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [X]  Pull request's target is not` master`
- [ ] Feature is working correctly on my local machine (if applicable)
- [X] Feature is working correctly on kitchen env. (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

File: plugins/taofurigana/plugin.js [hunk 1]
```diff
@@ -88,92 +88,110 @@ CKEDITOR.plugins.add('taofurigana', {
...
+					startNode = curRange.startContainer,
+					rubyElement,
+					rbElement,
+					rtElement,
+					range,
+					emptyElement,
+					rbHtml;
+				if (isInFigurana(startNode)) {
+					rubyElement = startNode.getAscendant('ruby');
+					console.log('rubyElement ', rubyElement);
```
Comment (marpesia) [inline plugins/taofurigana/plugin.js]: Please, remove the console.log

Comment (marpesia) [reply plugins/taofurigana/plugin.js]: Please remove the consoles
image

**Review by marpesia — CHANGES_REQUESTED** [review]
Pending the console.log remove

**Review by marpesia — APPROVED** [review]
LGTM

**Pull request merged**